src/Equinox/Tracing.fs

```diff
@@ -0,0 +1,107 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+module Tags =
+    // General tags
+
+    /// The full stream name
+    [<Literal>]
+    let stream_name = "eqx.stream_name"
```

Comment:
`let [<Literal>] stream_name` formatting would make this all far more legible  IMO

Reply:
already breaking the style guide by not using pascal case so, sure I guess

https://learn.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-literal-declarations

Reply:
gotta burn those innovation tokens (yes, pascalcase for literals is also critical for pattern matching too)